library/Fission/Types.hs

```diff
@@ -156,51 +172,38 @@ instance
           |> update Txt dnsLinkURL
           |> fmap \_ -> Right (AWS.DomainName baseURL)
 
-instance Has IPFS.Peer cfg => MonadLinkedIPFS (RIO cfg) where
-  getLinkedPeers = pure <$> Config.get
+instance MonadLinkedIPFS Fission where
+  getLinkedPeers =
```

Comment:
why not
```haskell
  getLinkedPeers = pure <$> asks ipfsRemotePeer
```
with short data flows like this, I think the one liners are much more readable

Reply:
@dholms it makes me so happy to hear you say that. You certainly have some Haskeller in you 😛 I agree with you, but am trying to not have what I'm used to get in the way of others.

I'm curious to have others weigh in. @bechurch / @icidasset?

Reply:
Haskell noob coming in: both feel the same really? 😅 

```haskell
getLinkedPeers = pure <$> asks ipfsRemotePeer
```
Is more haskelly and reads simpler once you know the operator.

```haskell
|> fmap pure
```
I think communicates more of whats going on under the hood. but it's atypical. (Haskell and those proficient in Haskell love their many operators)


My only preference is that we pick one overall, stick with it and reduce the number of operators you have to think about in any given file 😛. Again Haskell noob opinions over here.

Reply:
> whats going on under the hood

Yeah, we generally try to get away from under the hood. Subordinate detail to abstraction. I think that they're roughly equivalent in this case.

We did lose the visual guide by dropping `$`. `<$>` is `$` on containers `< >`. It's an extremely common operator, and I'm not sure why operators are scarier than named functions. You need to know what `bind` does, the same way that you need to know what `=<<` does. But I'm also coming from the far other end of the spectrum.

As an aside, I've been thinking about the number of operators since the last time it came up just before the break. It's interesting, since in C-family languages, they _heavily_ overload their operators, which should also be confusing. Is it just familiarity? I dunno. All things that I'm thinking about with Prime.